R/expect_lint.R

```diff
@@ -80,11 +80,12 @@ expect_lint <- function(content, checks, ..., file = NULL, language = "en") {
 
   local({
     itr <- 0L
+    this_env <- environment()
```

Comment:
Maybe call this `outer_env` everywhere for consistency? Also saves having to come up with new names all the time.

The inner code will be more clear too, I feel.
`outer_env$itr <- outer_env$itr + 1L`